modules/op-release-notes-1-0.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assembly:
+//
+// * openshift_pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-0_{context}"]
+= Release Notes for Red Hat {pipelines-title} Technology Preview 1.0
+
+[id="new-features-1-0_{context}"]
+== New features
+{pipelines-title} Technology Preview (TP) 1.0 is now available on {product-title} 4.4. {pipelines-title} TP 1.0 is updated to support:
+
+* Tekton Pipelines 0.11.3
+* Tekton `tkn` CLI 0.9.0
+* Tekton Triggers 0.4.0
+* ClusterTasks based on Tekton Catalog 0.11
+
+In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 1.0.
+
+=== Pipelines new features
+* Support for  v1beta1 API Version.
+* Support for an improved LimitRange. `limitRangeName` is now removed, and TaskRun now accounts for LimitRange on behalf of the user.
```

Comment:
@piyush-garg @nikhil-thomas - Could you please help us formulate this. Not clear about the meaning of this feature

Reply:
Previously, limitrange need to be specified in taskrun and pipelinerun, and tekton uses that limitrange's minimum, now it will see all the limit ranges in namespace and use the minimum, no need to specify

Reply:
as @piyush-garg said :wink:

Reply:
Support for an improved LimitRange. Previously, LimitRange was specified in the TaskRun and PipelineRun specifications, and the minimum of two LimitRange was used. Now there is no need to explicitly specify the LimitRange. The minimum LimitRange across the namespace is used.

Does this sound ok ?

Reply:
yes :ok_hand: